lib/rubocop/config.rb

```diff
@@ -172,8 +173,16 @@ def base_dir_for_path_parameters
 
     private
 
+    def validate_section_presence(name)
+      return unless @hash.key?(name) && @hash[name].nil?
+      fail ValidationError,
+           "empty section #{name} found " \
+           "in #{loaded_path || self}"
```

Comment:
Hi,

That expression is used at least 2 other times in that file.

The way I see things, I could factorize it in a method called loaded_path_or_self, but I'm not sure it will add something to the code comprehension.

@bbatsov, what's your input on this ?